src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -34,6 +38,11 @@ public string UniqueId
         /// </summary>
         public PackageLoader PackageLoader { get; private set; }
 
+        /// <summary>
+        ///   
+        /// </summary>
```

Comment:
Looks like we don't need a comment here? :)

Reply:
nice catch thanks.